### DISCUSSION TITLE
【分享】音频：PIKASONIC - Emptiness

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1533] [其他]
文件太大？扔gofile上！
![image](https://github.com/user-attachments/assets/67cc4704-c055-4896-8e67-8713cff33293)

[Gofile@jlHNTV - 70.6MB](https://gofile.io/d/jlHNTV)

音频信息：
![image](https://github.com/user-attachments/assets/7119c645-d542-460b-b029-31ee42c477bb)

**Comment by su2569** [comment]
6

**Comment by lllliufeng** [comment]
好家伙，装都不装了
